en/help.md

```diff
@@ -158,10 +159,36 @@ returned, even if the document produces a PDF with
 no errors. This can be specified by using `log` as the final token in
 the comment.
 
+---
+
+## HTML output (make4ht)
+
+If using the LaTeX online system, then an additional "TeX engine" may
+be used. `make4ht` may be specified, returning one or more HTML pages
+in the frame within the page. The command is configured to use
+`2,mathjax` configuration (see the [make4ht
```

Comment:
@josephwright  I don't know: we could certainly say more or less. Actually since I added the bit later about running it locally we could skip the entire configuration sentence.  The "2" bit explains why you get automatically added next/prev/up navigation and multi-page output, otherwise it would all be one file. But as I say that again in the next para thst could go here.